x-pack/solutions/security/plugins/security_solution/public/detections/links.ts

```diff
@@ -28,6 +31,61 @@ export const alertsLink: LinkItem = {
   title: ALERTS,
 };
 
+const alertsSubLink: LinkItem = {
+  ...alertsLink,
+  globalNavPosition: undefined,
```

Comment:
Do we need to pass `undefined` here or could we just no have the `globalNavPosition` property at all?